packages/components/src/components/list/list-entry/list-entry.tsx

```diff
@@ -0,0 +1,47 @@
+import { Component, h, Prop, Event, EventEmitter, Listen, Watch, Element } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-list-entry',
+  styleUrl: 'list-entry.scss',
+  shadow: true,
+})
+export class ListEntry {
+  @Element() host: HTMLElement;
+  @Prop({ mutable: true }) value: boolean;
+  @Prop() label: string;
+
+  @Prop() type: string;
+  @Event() ifxListEntryChange: EventEmitter;
+
+  @Watch('value')
+  valueChanged(newValue: boolean) {
+    if (newValue) {
+      this.host.setAttribute('value', 'true');
+    } else {
+      this.host.removeAttribute('value');
+    }
+  }
+
+  @Listen('ifxChange')
+  handleFilterEntryChange(event: CustomEvent) {
+    this.value = event.detail;
+    this.ifxListEntryChange.emit({ label: this.label, value: this.value, type: this.type });
+  }
+
+
+  render() {
+    return (
+      <div class="wrapper">
+        {this.type === 'checkbox' ? (
+          <div class="list-entry">
+            <ifx-checkbox size="s" value={this.value}>{this.label}</ifx-checkbox>
```

Comment:
this.value is set twice. Once inside @Watch, and once inside @Listen

Reply:
Setting the value in both @Listen and @Watch is necessary because they react to different triggers and perform different actions:

@Watch('value'):
The valueChanged method is a watcher for the value prop.
Whenever the value prop changes, regardless of how the change occurred, this watcher will run.
It's responsible for reflecting the new value prop to the host element's attribute, which might be important for CSS styling or other JS that queries the host element for its value attribute.

@Listen('ifxChange'):
The handleFilterEntryChange method is an event listener for the ifxChange event.
When ifxChange is emitted, presumably by the ifx-checkbox or ifx-radio-button, this method updates the value prop and emits a new event (ifxListEntryChange) to inform parent components of the change.